cypress/integration/progressIcons.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable scanjs-rules/identifier_localStorage, max-len */
+describe('The Heart Indicators', function() {
+    it('On the domain review, it sets the attributes on the heart icons correctly', function() {
+        cy.visit('http://localhost:1313/domains/identify/screen-for-substance-disorders/pre-assessment/');
+        cy.get('#q1-2').click();
+
+        cy.visit('http://localhost:1313/domains/identify/screen-for-substance-disorders/post-assessment/');
```

Comment:
`http://localhost:1313/domains/` could be put into a variable here to shorten some of these calls to cy.visit(). Looks good, though!

Reply:
Good point, I'll keep that in mind for the next tests.